src/Compilers/CSharp/Portable/xlf/CSharpResources.zh-Hans.xlf

```diff
@@ -454,12 +454,12 @@
       </trans-unit>
       <trans-unit id="ERR_GlobalUsingInNamespace">
         <source>A global using directive cannot be used in a namespace declaration.</source>
-        <target state="new">A global using directive cannot be used in a namespace declaration.</target>
+        <target state="translated">不能在命名空间声明中使用全局 using 指令。</target>
```

Comment:
Feels weird that "global" is localized but "using" is not. (I expect both to be not localized).

Reply:
@jonfortescue @wli3 could you please advise on what the appropriate channels are for localization feedback?

Reply:
Yes, file a Loc Issue [here](https://ceapex.visualstudio.com/CEINTL/_workitems/recentlyupdated/).

Reply:
Yes file a Loc Issue against the loc team [here](https://ceapex.visualstudio.com/CEINTL/_workitems/recentlyupdated/).